oneBarangay/static/assets/js/volt.js

```diff
@@ -260,96 +260,96 @@ d.addEventListener("DOMContentLoaded", function (event) {
-    });
+      plugins: [Chartist.plugins.tooltip()]
+    })
   }
 
-  if (d.getElementById("loadOnClick")) {
-    d.getElementById("loadOnClick").addEventListener("click", function () {
-      const button = this;
-      const loadContent = d.getElementById("extraContent");
-      const allLoaded = d.getElementById("allLoadedText");
+  if (d.getElementById('loadOnClick')) {
+    d.getElementById('loadOnClick').addEventListener('click', function () {
+      const button = this
+      const loadContent = d.getElementById('extraContent')
+      const allLoaded = d.getElementById('allLoadedText')
 
-      button.classList.add("btn-loading");
-      button.setAttribute("disabled", "true");
+      button.classList.add('btn-loading')
+      button.setAttribute('disabled', 'true')
 
       setTimeout(function () {
-        loadContent.style.display = "block";
-        button.style.display = "none";
-        allLoaded.style.display = "block";
-      }, 1500);
-    });
+        loadContent.style.display = 'block'
+        button.style.display = 'none'
+        allLoaded.style.display = 'block'
+      }, 1500)
+    })
   }
 
   const scroll = new SmoothScroll('a[href*="#"]', {
     speed: 500,
-    speedAsDuration: true,
-  });
+    speedAsDuration: true
+  })
 
-  if (d.querySelector(".current-year")) {
-    d.querySelector(".current-year").textContent = new Date().getFullYear();
+  if (d.querySelector('.current-year')) {
+    d.querySelector('.current-year').textContent = new Date().getFullYear()
   }
 
   // Glide JS
 
-  if (d.querySelector(".glide")) {
-    new Glide(".glide", {
-      type: "carousel",
+  if (d.querySelector('.glide')) {
+    new Glide('.glide', {
+      type: 'carousel',
       startAt: 0,
-      perView: 3,
-    }).mount();
+      perView: 3
+    }).mount()
   }
 
-  if (d.querySelector(".glide-testimonials")) {
-    new Glide(".glide-testimonials", {
-      type: "carousel",
+  if (d.querySelector('.glide-testimonials')) {
+    new Glide('.glide-testimonials', {
+      type: 'carousel',
       startAt: 0,
       perView: 1,
-      autoplay: 2000,
-    }).mount();
+      autoplay: 2000
+    }).mount()
   }
 
-  if (d.querySelector(".glide-clients")) {
-    new Glide(".glide-clients", {
-      type: "carousel",
+  if (d.querySelector('.glide-clients')) {
+    new Glide('.glide-clients', {
+      type: 'carousel',
       startAt: 0,
       perView: 5,
-      autoplay: 2000,
-    }).mount();
+      autoplay: 2000
+    }).mount()
   }
 
-  if (d.querySelector(".glide-news-widget")) {
-    new Glide(".glide-news-widget", {
-      type: "carousel",
+  if (d.querySelector('.glide-news-widget')) {
+    new Glide('.glide-news-widget', {
+      type: 'carousel',
       startAt: 0,
       perView: 1,
-      autoplay: 2000,
-    }).mount();
+      autoplay: 2000
+    }).mount()
   }
 
-  if (d.querySelector(".glide-autoplay")) {
-    new Glide(".glide-autoplay", {
-      type: "carousel",
+  if (d.querySelector('.glide-autoplay')) {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PrynsTag/oneBarangay/pullRequest?prid=7999497)
```suggestion
    if (d.querySelector(".glide-autoplay")) {
```